### PR TITLE
Refresh filebeat.yml with changes from libbeat.yml.

### PR DESCRIPTION
**Pull request by andrewkroh**
Closes elastic/libbeat#160

### CODE REVIEW

File: etc/filebeat.yml [hunk 1]
```diff
@@ -89,54 +90,169 @@ filebeat:
...
- # Uncomment the following if you want to ignore transactions created
- # by the server on which the shipper is installed. This option is useful
- # to remove duplicates if shippers are installed on multiple servers.
- # ignore_outgoing: true
-
-############################# Output ############################################
+############################# Output ##########################################
 
-# Configure what outputs to use when sending the data collected by filebeat.
+# Configure what outputs to use when sending the data collected by the beat.
```
Comment (urso) [inline etc/filebeat.yml]: s/the beat/filebeat/ ?

File: etc/filebeat.yml [hunk 2]
```diff
@@ -89,54 +90,169 @@ filebeat:
...
+  #tags: ["service-X", "web-tier"]
+
+  # Uncomment the following if you want to ignore transactions created
+  # by the server on which the shipper is installed. This option is useful
+  # to remove duplicates if shippers are installed on multiple servers.
+  #ignore_outgoing: true
+
+
+############################# Logging #########################################
+
```
Comment (urso) [inline etc/filebeat.yml]: logging looks pretty... scarce in comparison to other sections. Same for libbeat?

Comment (andrewkroh) [inline etc/filebeat.yml]: Same for libbeat. So we can update it in libbeat and push the changes to each of the beats. Automation would be nice so we don't have to manually update each beat.

File: etc/filebeat.yml (hunk 1 above)
Comment (andrewkroh) [inline etc/filebeat.yml]: Fixed. But I missed this change in topbeat and packetbeat.

File: etc/filebeat.yml (hunk 2 above)
Comment (monicasarbu) [inline etc/filebeat.yml]: Created an issue about the logging section: https://github.com/elastic/libbeat/issues/173

**Pull request merged**